File: aoc2025/java/src/test/java/com/peeekay/aoc2025/java/Day02Test.java
package com.peeekay.aoc2025.java;

import com.peeekay.aocCommon.AOCPuzzle;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class Day02Test {
    final AOCPuzzle day = new Day02(true);

    @Test
    void part1() {
        Assertions.assertEquals(1227775554L, day.part1());
    }

    @Test
    void part2() {
        Assertions.assertEquals(4174379265L, day.part2());
    }
}

File: aoc2022/kotlin/src/test/kotlin/Day02Test.kt
package com.peeekay.aoc2022.kotlin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

@DisplayName("Day 02")
internal class Day02Test {

    @Test
    fun `Part1 with test data`() {
        val day = Day02(isTest = true)
        val answer = day.partOne()

        assertEquals(15, answer)
    }

    @Test
    fun `Part2 with test data`() {
        val day = Day02(isTest = true)
        val answer = day.partTwo()

        assertEquals(12, answer)
    }
}